src/utils.js

```diff
@@ -58,6 +58,57 @@ export function getRootExpression(node) {
 }
 
 /*
+ * Returns the name of the reference node passed.
+ *
+ * example: var foo = document;
+ *  The node for foo will return 'document'
```

Comment:
I think the docs are poorly worded. This get's the _most recent_ reference to the node's value. So for insance
```javascript
var foo = window;
foo = document;
```
will return `document` rather than `window`.